.docker/docker-qgis-build.sh

```diff
@@ -84,7 +84,7 @@ echo "::endgroup::"
 #######
 echo "${bold}Building QGIS...${endbold}"
 echo "::group::build"
-${CTEST_BUILD_COMMAND}
+ctest -V -S /root/QGIS/.ci/config_build.ctest
```

Comment:
do you think we still the need the definition of `CTEST_BUILD_COMMAND` in `.docker/docker-variables.env` and in `.ci/config.ctest`?

Reply:
I don't know, I removed it. Will see if it fails without it

Reply:
It is definitely needed. I restored it